### PR TITLE
Cisco access points

### PR DESCRIPTION
**Pull request by francesco-noc**
I don't know why this time i had to add the .pre-commit-config.yaml file.
I had this error when I tryed to commit:
[ERROR] Your pre-commit configuration is unstaged.
`git add .pre-commit-config.yaml` to fix this.

### CODE REVIEW

**Comment by danner26** [comment]
The `weight` and `weight_unit` definitions should be available in the product datasheet. Can you please add those? Also add `airflow` if you can find if it is passive or directional. Thank you

**Comment by francesco-noc** [comment]
> @francesco-noc Is this ready for review again?

Yes.

File: device-types/Cisco/AIR-AP1815I-E-K9.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+weight: 400
+weight_unit: g
+airflow: passive
+is_full_depth: false
+comments: '[Cisco Aironet 1815i Access Point Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-1815-series-access-points/datasheet-c78-738243.html)'
+interfaces:
+  - name: GigabitEthernet0
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP1815I-E-K9.yaml:16]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP2702E-E-K9.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+weight_unit: kg
+airflow: passive
+comments: '[Cisco Aironet 2700 Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-2700-series-access-point/datasheet-c78-730593.html)'
+interfaces:
+  - name: GigabitEthernet0
+    type: 1000base-t
+    poe_mode: pd
+  - name: GigabitEthernet1
+    type: 1000base-t
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP2702E-E-K9.yaml:18]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP1702I-E-K9.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+weight_unit: kg
+airflow: passive
+comments: '[Cisco Aironet 1700 Series Access Point Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-1700-series/datasheet-c78-732347.html)'
+interfaces:
+  - name: GigabitEthernet0
+    type: 1000base-t
+    poe_mode: pd
+  - name: GigabitEthernet1
+    type: 1000base-t
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP1702I-E-K9.yaml:18]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-AP2802I-E-K9.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+weight_unit: kg
+airflow: passive
+comments: '[Cisco Aironet 2800 Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-2800-series-access-points/datasheet-c78-736497.html)'
+interfaces:
+  - name: GigabitEthernet0
+    type: 1000base-t
+    poe_mode: pd
+  - name: GigabitEthernet1
+    type: 1000base-t
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP2802I-E-K9.yaml:18]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-AP1832I-E-K9C.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+is_full_depth: false
+weight: 930
+weight_unit: g
+airflow: passive
+comments: '[Cisco Aironet 1830 Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-1830-series-access-points/datasheet-c78-735582.html)'
+interfaces:
+  - name: GigabitEthernet0
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP1832I-E-K9C.yaml:16]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-AP1832I-E-K9.yaml [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+weight: 930
+weight_unit: g
+airflow: passive
+is_full_depth: false
+comments: '[Cisco Aironet 1830 Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-1830-series-access-points/datasheet-c78-735582.html)'
+interfaces:
+  - name: GigabitEthernet0
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-AP1832I-E-K9.yaml:16]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP2702I-E-K9.yaml [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+weight_unit: kg
+airflow: passive
+comments: '[Cisco Aironet 2700 Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/aironet-2700-series-access-point/datasheet-c78-730593.html)'
+interfaces:
+  - name: GigabitEthernet0
+    type: 1000base-t
+    poe_mode: pd
+  - name: GigabitEthernet1
+    type: 1000base-t
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP2702I-E-K9.yaml:18]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP3702I-E-K9.yaml [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+u_height: 0
+is_full_depth: false
+weight: 1.13
+weight_unit: kg
+airflow: passive
+interfaces:
+  - name: GigabitEthernet0
+    type: 1000base-t
+    poe_mode: pd
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3702I-E-K9.yaml:15]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/AIR-CAP3702I-E-K9.yaml [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+    type: ieee802.11ac
```
Comment (DanSheps) [inline device-types/Cisco/AIR-CAP3702I-E-K9.yaml:16]: Missing console definition

File: device-types/Cisco/C9105AXI-E.yaml [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+slug: cisco-c9105axi-e
+part_number: C9105AXI-E
+u_height: 0
+weight: 329.5
+weight_unit: g
+airflow: passive
+is_full_depth: false
+comments: '[Cisco Catalyst 9105 Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9100ax-access-points/datasheet-c78-744062.html)'
+interfaces:
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/C9105AXI-E.yaml:13]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/C9105AXI-E.yaml [hunk 11]
```diff
@@ -0,0 +1,20 @@
+---
+manufacturer: Cisco
+model: Catalyst 9105AXI-E
```
Comment (DanSheps) [inline device-types/Cisco/C9105AXI-E.yaml:3]: Should include all variants of the 9105AXI (-A, -B, -H, -Q, -K, -T, -G, -D, -S, -I, -Z, -F, -N, -C, -R) if we are specifying the "-E".

File: device-types/Cisco/C9115AXI-E.yaml [hunk 12]
```diff
@@ -0,0 +1,22 @@
+---
+manufacturer: Cisco
+model: Catalyst 9115AXI-E
```
Comment (DanSheps) [inline device-types/Cisco/C9115AXI-E.yaml:3]: Should include all variants of the 9115AXI (-A, -B, -H, -Q, -K, -T, -G, -D, -S, -I, -Z, -F, -N, -C, -R) if we are specifying the "-E".

File: device-types/Cisco/C9120AXE-E.yaml [hunk 13]
```diff
@@ -0,0 +1,22 @@
+---
+manufacturer: Cisco
+model: Catalyst 9120AXE-E
```
Comment (DanSheps) [inline device-types/Cisco/C9120AXE-E.yaml:3]: Should include all variants of the 9120AXE (-A, -B, -H, -Q, -K, -T, -G, -D, -S, -I, -Z, -F, -N, -C, -R) if we are specifying the "-E".

File: device-types/Cisco/C9124AXE-E.yaml [hunk 14]
```diff
@@ -0,0 +1,22 @@
+---
+manufacturer: Cisco
+model: Catalyst 9124AXE-E
```
Comment (DanSheps) [inline device-types/Cisco/C9124AXE-E.yaml:3]: Should include all variants of the 9124AXE (-A, -B, -H, -Q, -K, -T, -G, -D, -S, -I, -Z, -F, -N, -C, -R) if we are specifying the "-E".

File: device-types/Cisco/C9115AXI-E.yaml [hunk 15]
```diff
@@ -0,0 +1,22 @@
...
+slug: cisco-c9115axi-e
+part_number: C9115AXI-E
+u_height: 0
+weight: 0.9
+weight_unit: kg
+airflow: passive
+is_full_depth: false
+comments: '[Cisco Catalyst 9115 Series Wi-Fi 6 Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9100ax-access-points/datasheet-c78-741988.html)'
+interfaces:
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/C9115AXI-E.yaml:13]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/C9120AXE-E.yaml [hunk 16]
```diff
@@ -0,0 +1,22 @@
...
+slug: cisco-c9120axe-e
+part_number: C9120AXE-E
+u_height: 0
+weight: 1.36
+weight_unit: kg
+airflow: passive
+is_full_depth: false
+comments: '[Cisco Catalyst 9120AX Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9120ax-series-access-points/datasheet-c78-742115.html)'
+interfaces:
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/C9120AXE-E.yaml:13]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

File: device-types/Cisco/C9124AXE-E.yaml [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+slug: cisco-c9124axe-e
+part_number: C9124AXE-E
+u_height: 0
+airflow: passive
+weight: 3.1
+weight_unit: kg
+is_full_depth: false
+comments: '[Cisco Catalyst 9124AX Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9100ax-access-points/nb-06-cat9124-ser-ap-ds-cte-en.html)'
+interfaces:
+  - name: WiFi
```
Comment (DanSheps) [inline device-types/Cisco/C9124AXE-E.yaml:13]: Should be "Dot11Radio0"

There should also be a "Dot11Radio1"

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Comment by francesco-noc** [comment]
Changes made!

**Review by DanSheps — APPROVED** [review]

**Comment by danner26** [comment]
@francesco-noc please resolve the conflicts or close this PR, thank you.

**Comment by francesco-noc** [comment]
Sorry, I can work on it (and on other PRs) tomorrow.

**Pull request closed without merging**